docs/_includes/documentation-menu.html

```diff
@@ -8,7 +8,6 @@
 			<li><a href="{{docu}}/concepts/items.html">Items</a></li>
 			<li><a href="{{docu}}/concepts/things.html">Things</a></li>
 			<li><a href="{{docu}}/concepts/discovery.html">Inbox & Discovery</a></li>
-			<li><a href="{{docu}}/concepts/events.html">Events</a></li>
```

Comment:
As the entry is removed from the `Concepts` shouldn't we add one to the `Features`?